pkg/landscaper/installations/imports/validation.go

```diff
@@ -347,14 +347,30 @@ func (v *Validator) checkStateForParentImport(fldPath *field.Path, importName st
 	return nil
 }
 
+type IsExportingDataFunc func(*installations.InstallationBase, string) bool
```

Comment:
can you add a docstring explaining the function

Reply:
I can add a docstring.
However, since it is used only internally, should it be begin lowercase to avoid exporting?

Reply:
then I would propose to do both. so make it private and add a string what the boolean means

Reply:
done